requirements-dev.txt

```diff
@@ -2,6 +2,7 @@ coverage==5.1
 django-compressor==2.1.1
 django-debug-toolbar==1.11                      # https://github.com/jazzband/django-debug-toolbar
 django-debug-toolbar-request-history==0.0.11    # for debugging database queries on async requests.
+future==0.17.1                                  # for Python2 and 3 compatibility
```

Comment:
you import `future` in `external_api.py` which is a real code file not a test file, so this needs to go in the main `requirements.txt` file

Reply:
Moved it to the requirements.txt